support-frontend/assets/pages/[countryGroupId]/checkout.tsx

```diff
@@ -280,29 +256,22 @@ const stripePublicKey = getStripeKey(
 );
 
 export function Checkout() {
-	const [productCatalog, setProductCatalog] = useState<ProductCatalog>();
-
-	useEffect(() => {
-		void fetch('/api/products')
```

Comment:
I guess we will delete the server side endpoint once no client side are using the old code any more

Reply:
I thought about doing it as part of this PR, but I found it useful to reference. But maybe I should just use standard URL: https://product-catalog.guardianapis.com/product-catalog.json

Reply:
I'm going to remove it - good shout

Reply:
actually I was assuming it was intentional in case of a race condition - i.e. someone loaded the old client side code, but the new server side is already responding to the request.  But as a one off it's probably going to be ok, assuming that information is only fetched on initial page load.